src/Tests/dotnet-new.Tests/DotnetNewCommandTests.cs

```diff
@@ -1,64 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Threading.Tasks;
-using FluentAssertions;
-using Microsoft.NET.TestFramework;
-using Microsoft.NET.TestFramework.Assertions;
-using Microsoft.NET.TestFramework.Commands;
-using Microsoft.TemplateEngine.TestHelper;
-using VerifyTests;
-using VerifyXunit;
-using Xunit;
-using Xunit.Abstractions;
-
-namespace Microsoft.DotNet.New.Tests
-{
-    [UsesVerify]
-    [Collection("Verify Tests")]
-    public class DotnetNewCommandTests : SdkTest
-    {
-        private readonly ITestOutputHelper _log;
-
-        public DotnetNewCommandTests(ITestOutputHelper log) : base(log)
-        {
-            _log = log;
-        }
-
-        [Fact]
-        public Task CanShowBasicInfo()
-        {
-            string home = TestUtils.CreateTemporaryFolder("Home");
-            string workingDirectory = TestUtils.CreateTemporaryFolder();
-
-            var commandResult = new DotnetNewCommand(_log)
-                .WithCustomHive(home)
-                .WithWorkingDirectory(workingDirectory)
-                .Execute();
-
-            commandResult.Should()
-                .ExitWith(0).And.NotHaveStdErr();
-
-            return Verify(commandResult.StdOut).UniqueForOSPlatform();
-        }
-
-        [Fact]
-        public void CanShowFullCuratedList()
```

Comment:
Did we drop this test? I cannot see it anywhere in PR.

Reply:
Yes, now it is exactly the same as ShowBasicInfo test.